src/cashweb/relay/index.ts

```diff
@@ -520,6 +521,9 @@ export class RelayClient extends ReadOnlyRelayClient {
         if (item.type === 'text') {
           return this.messageConstructor.constructTextEntry({ ...item })
         }
+        if (item.type === 'forward') {
```

Comment:
Good, we probably just need to factor these out somewhere nice so it's easier to implement them.